test/e2e-helm/e2e_helm_olm_test.go

```diff
@@ -15,25 +15,72 @@
 package e2e_helm_test
 
 import (
+	"os/exec"
+	"path"
 	"path/filepath"
+	"strings"
 
 	. "github.com/onsi/ginkgo" //nolint:golint
+	. "github.com/onsi/gomega" //nolint:golint
 
 	testutils "github.com/operator-framework/operator-sdk/test/internal"
 )
 
-var _ = PDescribe("Integrating Helm Projects with OLM", func() {
+var _ = Describe("Integrating Helm Projects with OLM", func() {
 	Context("with operator-sdk", func() {
+		const operatorVersion = "0.0.1"
+
 		BeforeEach(func() {
-			By("Turning off interactive prompts for all generation tasks.")
+			By("turning off interactive prompts for all generation tasks.")
 			replace := "operator-sdk generate kustomize manifests"
 			testutils.ReplaceInFile(filepath.Join(tc.Dir, "Makefile"), replace, replace+" --interactive=false")
 		})
 
 		AfterEach(func() {
+			By("destroying the deployed package manifests-formatted operator")
+			cleanupPkgManCmd := exec.Command(tc.BinaryName, "cleanup", "packagemanifests",
+				"--operator-version", operatorVersion,
+				"--timeout", "4m")
+			_, _ = tc.Run(cleanupPkgManCmd)
 		})
 
-		It("Should allow generate the OLM bundle and run it", func() {
+		It("should generate and run a valid OLM bundle and packagemanifests", func() {
+			By("building the bundle")
+			err := tc.Make("bundle")
+			Expect(err).NotTo(HaveOccurred())
+
+			By("validating the bundle")
+			bundleValidateCmd := exec.Command(tc.BinaryName, "bundle", "validate", "./bundle")
+			_, err = tc.Run(bundleValidateCmd)
+			Expect(err).NotTo(HaveOccurred())
+
+			By("building the operator bundle image")
+			// Use the existing image tag but with a "-bundle" suffix.
+			imageSplit := strings.SplitN(tc.ImageName, ":", 2)
+			bundleImage := path.Join("quay.io", imageSplit[0]+"-bundle")
+			if len(imageSplit) == 2 {
+				bundleImage += ":" + imageSplit[1]
+			}
+			err = tc.Make("bundle-build", "BUNDLE_IMG="+bundleImage)
+			Expect(err).NotTo(HaveOccurred())
+
+			By("loading the project image into Kind cluster")
+			err = tc.LoadImageToKindClusterWithName(bundleImage)
+			Expect(err).Should(Succeed())
+
+			By("generating the operator package manifests")
+			err = tc.Make("packagemanifests")
+			Expect(err).NotTo(HaveOccurred())
+
+			// fixme: packagemanifests is not running/deploying the controller
+			// see that it just pass in the tests for go because the project/manager is deployed already
+			//By("running the package")
+			//runPkgManCmd := exec.Command(tc.BinaryName, "run", "packagemanifests",
+			//	"--install-mode", "AllNamespaces",
+			//	"--operator-version", operatorVersion,
+			//	"--timeout", "4m")
+			//_, err = tc.Run(runPkgManCmd)
+			//Expect(err).NotTo(HaveOccurred())
```

Comment:
Hi @estroz and @joelanford, 

Now that we split the tests properly was possible to identify that if the project is not deployed on the cluster the  `run packagemanifests` is never succeed.

Reply:
I discovered what are the pre-requirements to make it work. Solved.